libs/fn/src/lib/list/selectable-item/selectable-item.directive.ts

```diff
@@ -3,11 +3,16 @@ import {
     ChangeDetectorRef,
```

Comment:
for further: i would ask you to split changes to do not have several components updated in the same pr if it's not about refactoring. this pr about the cdk so we can review & merge it then start using cdk in these components.

it's my opinion, we can discuss it if needed.